main.js

```diff
@@ -53,7 +55,7 @@ clearButton.addEventListener('click', clearInputs);
 submitButton.addEventListener('click', updateLatestGuesses);
 resetButton.addEventListener('click', resetGame);
 // Winner Card
-closeButton.addEventListener('click', closeWinnerCard);
+// closeButton.addEventListener('click', closeWinnerCard);
```

Comment:
We can solve the close button problem with event bubbling.  I will add this tonight.